diagrams/credential.svg

```diff
@@ -1 +1,18 @@
\ No newline at end of file
+<svg version="1.1" viewBox="-5 -5 330 245" xmlns="http://www.w3.org/2000/svg">
+<style><![CDATA[
+	rect {stroke:#000;stroke-width:1uu;}
+]]></style>
+
+<rect fill="#bad" width="320" height="230" rx="25"/>
+
+<text style="font-family:sans-serif;font-size: 1.5em" x="60" y="40">Verifiable Credential</text>
+
+<rect fill="#dab" x="45" y="65" width="240" height="40" rx="4"/>
+<text x="100" y="90">Credential Metadata</text>
+
+<rect fill="#bda" x="45" y="120" width="240" height="40" rx="4"/>
```

Comment:
Color doesn't ~match orig.

Reply:
Yup. Because elsewhere, *sometimes* the claim information is presented with the same background as the proof. Should we move to consistently distinguishing claims and proof by background?

In bd4f6b3 I matched the background colour of a verifiable credential in credential.svg to the background colour used for a verifiable credential used in presentation.svg.

But I think it would be better to match the colours used in zkp-cred-pres.svg applying them to both credential.svg and presentation.svg. This should probably also apply to credential.graph and presentation.graph too.

In any event, changing colours in those files is pretty trivial to get exact matches if we want them - whether for consistency or because we decide we like some other scheme. 

I don't care what colours we use, I think it is important that they are consistent across our diagrams, and I think it is critical that we do a decent contrast check on them. (Which I have not yet done).